XamarinCommunityToolkit/Views/CameraView/FormsCameraView.ios.cs

```diff
@@ -0,0 +1,553 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using AVFoundation;
+using CoreGraphics;
+using CoreMedia;
+using Foundation;
+using UIKit;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public sealed class FormsCameraView : UIView, IAVCaptureFileOutputRecordingDelegate
```

Comment:
Just wondering if we need to make this class `sealed`. Since this is a `renderer` I think that we should allow the dev to extend this class. What do you think?